utilities/test_suite/HOST/Tensor_host.cpp

```diff
@@ -1220,13 +1257,6 @@ int main(int argc, char **argv)
                     else
                         print_array(static_cast<Rpp16f *>(reductionFuncResultArr), reductionFuncResultArrLength, precision);
                 }
-                else if (dstDescPtr->dataType == RpptDataType::F32)
```

Comment:
I hope this deletions and additions are not breaking the previous code added for sum
Please check once @snehaa8

Reply:
Yes confirmed these changes don't affect any existing functionality